client/src/components/ProjectSettings.vue

```diff
@@ -120,6 +124,25 @@ export default defineComponent({
         />
       </v-flex>
     </v-layout>
+    <v-layout>
+      <v-tooltip top>
+        <template v-slot:activator="{ on, attrs }">
+          <div
+            v-bind="attrs"
+            v-on="on"
+          >
+            <v-switch
+              v-model="globalImportExport"
+              @click="changed = true"
```

Comment:
I tend to think that on a form, toggles should perform automatic saves. Unlike text fields where you can't tell when the user is finished, a toggle is a one-and-done action like a button. Perhaps the change function should call the save function instead of making the Save button enabled.

Reply:
I think that would be less intuitive, considering the proximity of the save button which enables immediately after clicking the toggle. Having text fields that require a manual save and a toggle which doesn't seems inconsistent to me.

Reply:
Okay, fair enough. I just thought it was worth mentioning.